src/pynwb/ndx_microscopy/testing/_mock.py

```diff
@@ -152,8 +161,8 @@ def mock_Segmentation2D(
 
```

Comment:
for the signature of mock_Segmentation2D, summary_images is now `Optional[List[ndx_microscopy.SummaryImage]]` right?

Reply:
https://github.com/catalystneuro/ndx-microscopy/pull/42/files#diff-c2db3241d880fe1cf9aa30c065e243fca5714c60828eee717264df3f5e248f31R150-R158